lookout/style/typos/preprocessing.py

```diff
@@ -0,0 +1,35 @@
+from typing import Set, Union
+
+import pandas
+
+from lookout.style.typos.utils import Columns
+
+
+def check_split(split: str, tokens_set: Set[str]) -> bool:
+    """
+    Check whether all elements of the split are in tokens_set.
+
+    :param split: String of space-separated tokens.
+    :param tokens_set: Set of tokens (reference vocabulary).
+    :return: True when all tokens of split belong to tokens_set.
+    """
+    if type(split) != str:
```

Comment:
```suggestion
    if not isinstance(str, split):
```
But you specify the type as str in function header: `split: str` so that means that you do not expect other types. 
You should remove this condition or fix type annotation.